annotator-core/src/main/java/edu/ucr/cs/riple/core/metadata/field/FieldDeclarationStore.java

```diff
@@ -123,6 +145,22 @@ public ImmutableSet<String> getInLineMultipleFieldDeclarationsOnField(
     return inLineGroupFieldDeclaration.orElse(ImmutableSet.copyOf(fields));
   }
 
+  /**
+   * Returns true if the given field is declared in the given class but not initialized at
+   * declaration. A field declaration can contain multiple inline field declarations. This method
+   * will return true if any of the declared fields is not initialized.
+   *
+   * @param field Location of field to check.
+   * @return True if at least on of the given declarations at the given location is not not
+   *     initialized.
+   */
+  public boolean isUninitializedField(OnField field) {
+    // According to javadoc, Multimap.get() returns an empty collection if key is not found,
+    // therefore,
+    // we do not need to check for key existence.
```

Comment:
formatting :-)

Reply:
Sorry again 😞 fixed it in 1646c50e660b7a45eeefbb492d21f9a5725a0033